### ISSUE TITLE
Think of a good Issue labelling system

### ISSUE THREAD

**Issue by bemayr** [uncodeference/unmeetup#7]
Probably combine/take inspiration from
- https://github.com/WrocTypeScript/talks/labels
- https://github.com/cyclejs/cyclejs/labels

What ideas do you have?

**Comment by juanluisgv1** [comment]
I think the guys from [WrocTypeScript](https://github.com/WrocTypeScript/talks/labels) achieve a good labelling system, as you already mentioned in #3 

- Duplicate
- Idea
- Lightning talk ⚡️
- Proposal
- Up for grabs

**Comment by bemayr** [comment]
Yeah, indeed it's nice, i just combined it with a "tree-like-structure" and set it up. Let's present them today.
